### PR TITLE
update jenkinsfile for benchmarks

### PR DESCRIPTION
**Pull request by MonssafToukal**

### CODE REVIEW

**Comment by MonssafToukal** [comment]
I ran the benchmarks on a fork of this repo and I encountered this error: https://gist.github.com/JSOBot/7d2d6abe356134cec9b47cb5322c8b39

**Comment by dpo** [comment]
1. Aren't you also supposed to remove the Jenkinsfile in the top folder?
2. who tries to install Conda? Is it Plots.jl?

File: benchmark/Jenkinsfile [hunk 1]
```diff
@@ -56,34 +57,56 @@ pipeline {
...
+            git checkout main
+            git pull origin main
+            git fetch origin
+            LOCAL_BRANCH_NAME="temp_bmark"
+            git branch -D $LOCAL_BRANCH_NAME || true
+            git fetch origin pull/$pullrequest/head:$LOCAL_BRANCH_NAME
+            git checkout $LOCAL_BRANCH_NAME --
           '''
         }
+        sh "git clone https://github.com/ProofOfConceptForJuliSmoothOptimizers/BenchmarkSetup.git || true"
```
Comment (dpo) [inline benchmark/Jenkinsfile:90]: org name here

Comment (MonssafToukal) [reply benchmark/Jenkinsfile:90]: At the moment, this repository contains all the scripts that run the benchmarks.

**Comment by amontoison** [comment]
Sorry @MonssafToukal, I didn't see your PR...
I tried to fix the problem with #110, I just reused what you did for Krylov.jl.

**Comment by dpo** [comment]
Can this be closed?

**Comment by MonssafToukal** [comment]
Yes, it can be closed now.

**Pull request closed without merging**